d9/.ci/deploy/pantheon/dev-multidev

```diff
@@ -38,8 +38,12 @@ terminus -n drush "$TERMINUS_SITE.$TERMINUS_ENV" -- cr
 terminus -n env:clear-cache $TERMINUS_SITE.$TERMINUS_ENV
 
 # Ensure secrets are set
-terminus -n secrets:set "$TERMINUS_SITE.$TERMINUS_ENV" token "$GITHUB_TOKEN" --file='github-secrets.json' --clear --skip-if-empty
-
+if [[ ! -z "$GH_TOKEN" ]]
```

Comment:
Why do we have both `$GH_TOKEN` and `$GITHUB_TOKEN`?

These blocks could be less verbose:
```
terminus -n secrets:set "$TERMINUS_SITE.$TERMINUS_ENV" token "${GH_TOKEN:-$GITHUB_TOKEN}" --file='.build-secrets/tokens.json' --clear --skip-if-empty
```

Reply:
GITHUB_TOKEN is the name we normally use with every CI provider; however, it's set by default with Github Actions to a read-only token for the current repo. Then, we set a new GH_TOKEN that should be used as secret in this scenario